templates/events/manage_event.html

```diff
@@ -14,12 +14,50 @@ <h1 class="pg-title">{{ title }}</h1>
         <div>
             {% block pre_form %}
             {% endblock pre_form %}
-            <form method="post" class="my-2" {% include "generic/attrs.html" with attrs=form_attrs %} x-data="{ type: '{{ secondary_key }}' }">
+            <form method="post" class="my-2" {% include "generic/attrs.html" with attrs=form_attrs %}
+                  x-data="eventForm('{{ secondary_key }}', '{{ trigger_form.initial.type }}')"
+                  x-init="watchTriggerType()">
                 {% csrf_token %}
                 {% block form %}
                     <h1 class="pg-title">Event Details</h1>
                     {% render_form_fields trigger_form %}
                     {% render_form_fields action_form.primary %}
+                    <script>
```

Comment:
Nit: Can we put this script at the bottom? It will make the code look cleaner

Reply:
You can use the [reusable data](https://alpinejs.dev/directives/data#re-usable-data) pattern as well